dev/Repeater/TestUI/Samples/ItemTemplateSamples/DisposableUserControl.xaml.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Threading;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The User Control item template is documented at https://go.microsoft.com/fwlink/?LinkId=234236
+
+namespace MUXControlsTestApp.Samples
+{
+    public sealed partial class DisposableUserControl : UserControl
+    {
+
+        private int num;
```

Comment:
why is this needed ?

Reply:
That was from @chingucoding's branch. Looks like it's not really needed since we now understand that custom properties don't come into play. I can remove the unnecessary bits.

Reply:
I'm guessing we dont need the custom user control at all ?

Reply:
We need something to be able to track that we leaked one of the objects. So either a custom user control or custom property would do it. The custom UserControl was what started the discussion so I like that as a repro scenario.

Reply:
Right, we need something to track, but don't need it to be a user control with markup. I was thinking something like below but both work fine. 
```
class LeakTrackingContainer: ContentControl
{
    ctor(){}
    dtor{}
    static .. instanceCount
    bool IsLeaking()
    {
        return instanceCount != 0
    }
}
```

Reply:
Sure, that would work too, but i'll keep it as-is.